statusboard/src/pages/Projects/Projects.tsx

```diff
@@ -128,11 +129,13 @@ function Projects(): JSX.Element {
             />
           )}
           <br />
+          <div className="hide3">
```

Comment:
Nitpick: Give more meaningful name. Since everyone are using auto-completion, even naming it "hideThirdColumn" would be fine, probably more Java than JavaScript but still :)